app/helpers/users_helper.rb

```diff
@@ -1,2 +1,10 @@
 module UsersHelper
+  def sorting_link(column, table)
+    sorting_order = (params[:column] == column) ? toggle_order(params["#{table}_sorting"]) : "ASC"
```

Comment:
Can we make reusable code for pagination and sorting? As mentioned earlier, talk with Bharat/Gautam.